ui/viewers/nuxeo-image-viewer.js

```diff
@@ -78,6 +78,14 @@ import '../nuxeo-icons.js';
           value: false,
         },
 
+        /**
+         * A short text alternative for the image.
+         */
+        alt: {
+          type: String,
+          value: null,
```

Comment:
Would it make a difference if we removed "value: null," ? If not, I'd probably remove it.

Reply:
I was replicating how [iron-image implements it](https://github.com/PolymerElements/iron-image/blob/master/iron-image.js#L145). I will remove it 👍

Reply:
Indeed. Not sure why they do it, though 🤔

Reply:
After doing some testing around this, I couldn't find any reason to have the default value set as null. I removed it.